coding-the-legislation/reforms.md

```diff
@@ -35,28 +38,79 @@ A reform that modifies a formula (such as our `income_tax_reform` example) is ca
 
 ### Parametric reforms
 
-Unlike *structural reforms*, *parametric reforms* do not directly modify any formula: they apply changes to legislation parameters only.
+A reform that apply changes to legislation parameters is called a *parametric reforms*. 
```

Comment:
A reform that apply changes only to the legislation parameters is called a *parametric reforms*.